miso-web/src/main/webapp/pages/editRun.jsp

```diff
@@ -256,9 +256,22 @@
         </thead>
         <tbody>
         <tr>
-          <td><fmt:formatDate pattern="dd/MM/yyyy" value="${run.startDate}"/></td>
           <c:choose>
-          <c:when test="${(run.health.isDone() and empty run.completionDate)}">
+          <c:when test="${run.id == 0 or fn:contains(SPRING_SECURITY_CONTEXT.authentication.principal.authorities,'ROLE_ADMIN')}">
+              <td><form:input path="startDate"/></td>
+          
+              <script type="text/javascript">
+              Utils.ui.addDatePicker("startDate");
+              </script>
+            </c:when>
+            <c:otherwise>
+              <td id="startDate">
+                <fmt:formatDate pattern="dd/MM/yyyy" value="${run.startDate}"/>
+              </td>
+            </c:otherwise>
+          </c:choose>
+          <c:choose>
+          <c:when test="${(run.health.isDone() and empty run.completionDate) or run.id == 0 or fn:contains(SPRING_SECURITY_CONTEXT.authentication.principal.authorities,'ROLE_ADMIN')}">
```

Comment:
I don't know that completion date should be editable (even by admins) before the run is actually complete. What do you think?